server/notification-providers/grafana-oncall.js

```diff
@@ -0,0 +1,54 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class GrafanaOncall extends NotificationProvider {
+
+    name = "GrafanaOncall";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+        try {
+
+            // If heartbeatJSON is null, assume we're testing.
+            if (heartbeatJSON == null) {
+                let GrafanaOncallTestData = {
+                    title: "This is a test",
+                    message: msg,
+                    state: "alerting",
+                };
+                await axios.post(notification.grafanaoncallURL, GrafanaOncallTestData);
+                return okMsg;
+            }
+
+            if (heartbeatJSON["status"] === DOWN) {
+                let grafanadowndata = {
+                    title: monitorJSON["name"] + " is down",
+                    message: heartbeatJSON["msg"],
+                };
+                await axios.post(
+                    notification.grafanaoncallURL,
+                    grafanadowndata
+                );
+                return okMsg;
+            } else if (heartbeatJSON["status"] === UP) {
+                let grafanaupdata = {
+                    title: monitorJSON["name"] + " is up",
+                    message: heartbeatJSON["msg"],
+                    alert_uid: "",
```

Comment:
Is there any point in including this, given that it is just an empty string and the docs seem to indicate that it is not required?
```suggestion
```

Reply:
I should have removed it before commit, i was using it to try to follow the documentation from oncall who specify the alert_uid for one 'outage' but since uptime kuma doesn't have any information about this, i'll remove it.